Dockerfile

```diff
@@ -0,0 +1,48 @@
+### Base image...
+FROM python:3-slim-buster as Base
+
+# Install poetry...
+RUN pip install poetry
+
+# Copy application code...
+COPY . .
+
+# Install prerequisites
+RUN poetry install --no-dev --no-root
+
+# Expose port to listen on...
+EXPOSE 8000
+
+
+
+### Production image...
+FROM base as production
+
+# Install gunicorn...
+RUN pip install gunicorn
+
+# Define entry point...
+ENTRYPOINT poetry run gunicorn --bind 0.0.0.0 "todo_app.app:create_app()"
+
+### Commands to build and run development image
+# docker build --target development --tag todo-app:dev .
+# docker run -d -p 7000:8000 --env-file .\.env --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+
+### Container should then be running on http://localhost:7000/
```

Comment:
I'd be tempted to pull these out of this file and just leave them in the Readme - comments like this are classic places that tend to get ignored when people make changes and get outdated very quickly!